wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -640,13 +652,15 @@ <h2>Dropdown buttons</h2>
                         <div data-dropdown-toggle="" class="o-icon  c-dropdown__toggle  [ icon icon-arrow-down ]"></div>
                     </a>
                     <div class="t-dark">
-                        <ul role="menu" class="c-dropdown__menu  u-toggle  u-arrow u-arrow--tl u-background">
-                            <li class="c-dropdown__item ">
-                                <a href="/admin/pages/2/move/" class="u-link is-live ">
-                                    Move
-                                </a>
-                            </li>
-                        </ul>
+                        <nav aria-label="Dark inline dropdown menu example">
```

Comment:
```suggestion
                        <nav aria-label="Dark inline dropdown menu example" role="navigation">
```

Reply:
As per previous comments, I won't add the `role` attribute.